virttest/utils_test/libvirt.py

```diff
@@ -2371,19 +2372,47 @@ def create_tpm_dev(params):
     return tpm_dev
 
 
-def get_vm_device(vmxml, dev_tag, index=0):
+def get_vm_device(vmxml, dev_tag, dev_dict=None, index=0):
```

Comment:
How about 
```suggestion
def get_vm_device(vmxml, dev_tag, index=0, dev_dict=None):
```
Though I didn't find reference of this function in tp-libvirt, just to be safe, adding the new arg to the end of the list could reduce the risk. What do you say?

Reply:
Great ,Updated like this can avoid some potential risk ,Thanks your comment.
Updated.